app/src/js/utils/status.js

```diff
@@ -12,3 +12,9 @@ export const pdrStatus = {
   Completed: 'completed',
   Failed: 'failed'
 };
+
+export const reconciliationReportStatus = {
+  Generated: 'Generated',
+  Pending: 'Pending',
+  Failed: 'Failed'
```

Comment:
I think lauren wanted us to use 'conflict' instead of failed. unless failed is the inability to run the report?  So 'conflict' is we generated the report and files don't match, and 'failed' is we somehow were unable to generate the report?

Reply:
UI/UX input: This is regarding the status of the report as it is created like we do when a granule is ingested. We need to validate if the report was generated or not. What you are referring to is regarding an individual report after it's generated and the status is the file comparison and missing files. I think we should change the term on the individual report itself from "status" to "state" or something else for less confusion. I'll think on that and propose to OXWG to see what they say.

Reply:
I like state better. 👍

Reply:
Yeah, it's somehow confusion.  I like 'state' also.

Reply:
We should definitely stick to correct words here:  status => (passed/conflict), state => (generated/pending/failed).

Although the tenses agree better with,  pass/conflict, I still like passed/conflict better. but what do I know.

Reply:
@jjmccoy will you get back to us with the OXWG decision on that? We can write a ticket to ammend.  I will approve the rest of the current changes.

Reply:
Okay will do.